src/sensai/featuregen/feature_generator.py

```diff
@@ -32,23 +32,25 @@ class FeatureGenerator(ToStringMixin, ABC):
     def __init__(self,
             categorical_feature_names: Optional[Union[Sequence[str], str]] = None,
             normalisation_rules: Sequence[data_transformation.DFTNormalisation.Rule] = (),
-            normalisation_rule_template: data_transformation.DFTNormalisation.RuleTemplate = None,
-            add_categorical_default_rules=True):
+            normalisation_rule_template: Optional[data_transformation.DFTNormalisation.RuleTemplate] = None,
+            add_categorical_default_rules: bool = True):
         """
         :param categorical_feature_names: either a sequence of column names or a regex that is to match all categorical feature names
-            (which must not only work for the feature generated by this feature generator, i.e. it should not match feature names generated
+            (which must not only work for the feature generated by this feature generator, i.e., it should not match feature names generated
             by other feature generators).
             It will be ensured that the respective columns in the generated data frames will have dtype 'category'.
-            Furthermore, presence of meta-information can later be leveraged for further transformations, e.g. one-hot encoding.
-        :param normalisation_rules: Rules to be used by DFTNormalisation (e.g. for constructing an input transformer for a model).
+            Furthermore, the presence of meta-information can later be leveraged for further transformations, e.g., one-hot encoding.
```

Comment:
In British English, "e.g." and "i.e." are not followed by a comma.
https://superior-english.com/en/2017/08/08/comma-after-i-e-and-e-g/

Also, note that commas must always be followed by a space. But please remove the commas you added.